build_release.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+cargo rustc --release
```

Comment:
I am curious, what is the difference between ````cargo build --release```` and ````cargo rustc --release````?

Reply:
There is none :D - I was trying out ```cargo rustc --release -- -C prefer-dynamic``` but realized that dynamic linking is not a good idea. Will change it back with the next commit